compiler-rt/test/asan/TestCases/Posix/no_asan_gen_globals.c

```diff
@@ -3,7 +3,10 @@
 // Make sure ___asan_gen_* strings do not end up in the symbol table.
 
 // RUN: %clang_asan %s -o %t.exe
-// RUN: nm %t.exe | FileCheck %s
+// RUN: nm %if target={{.*aix.*}} %{ -X32_64 %} %t.exe | FileCheck %s
```

Comment:
Change `nm` to `%nm` and set the LIT config to have it expand to `nm -X32_64` on AIX.